configextractor.py

```diff
@@ -167,22 +169,24 @@ def classification_checker(res_section, parser_name, file_parsers):
 
 
 def subsection_builder(parent_section: ResultSection = None, fields: dict = {}):
+    # TODO: maybe iterate through fields and check if field in FIELD_TAG_MAP? not sure which is smaller
     for field, tag in FIELD_TAG_MAP.items():
         if field in fields:
             table_body = []
             table_section = ResultSection(f"Extracted {field.capitalize()}")
             field_data = fields[field]
             if tag:
+                # TODO: is field_data always a list?
```

Comment:
it's either a list or a list of lists

Reply:
Okay cool, and I'm assuming if it's a list of lists then we want the nested lists to be tagged per nested list and not per item in the nested list ex. `"network.static.uri": (a, b, c) versus "network.static.uri": a, "network.static.uri": b, "network.static.uri": c`? Also we should add a check here that checks if the`tag` exists in `table_section` already, and then not try to add it if it does.

Reply:
so right now we aren't tagging anything that's a listofstringtuples. all the values in FIELD_TAG_MAP are listofstrings. Also sometimes it doesn't really make much sense to tag them because values in listofstringtuples are related but different and so don't really belong under one tag. For example  
https://github.com/Defense-Cyber-Crime-Center/DC3-MWCP/blob/0d27dea113751ef1aff727aaa2cf13c709e6beab/mwcp/config/fields.json#L252
As well AL already does the deduplication for tags. Would it improve readability if we explicitly add a check?

Reply:
Okay good to know.

Valid point.

Fair, us explicitly checking if the tag exists would be the same efficiency as calling add_tag from what I can tell, so we should be good with status quo. 

Resolving convo.